editor/components/rich-text/index.js

```diff
@@ -723,9 +738,12 @@ export class RichText extends Component {
 	}
 
 	componentDidUpdate( prevProps ) {
+		if ( ! this.editor ) {
```

Comment:
Looks like this can be moved back to minimise diff.

Reply:
Fixed in https://github.com/WordPress/gutenberg/pull/5794/commits/f4a4358dbfeba0e5a1770a1632a5c5d920ccca91